### ISSUE TITLE
Empty gaps seen in the site editor.

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/twentytwentytwo#272]
**Describe the bug**
I open the Site Editor and add a paragraph block. Add a black background color. Notice there is an empty space under the header. 

**To Reproduce**
1. WP 5.9 beta 1. Go to Site Editor.
2. Add a couple of paragraph blocks with a black background.
3. Notice the large gap just below the header and the smaller gaps between the paragraph blocks.

![Screenshot 2021-12-03 at 09 27 32](https://user-images.githubusercontent.com/5323259/144570210-a7bca1a4-1940-4e13-ab32-6680523c39e5.jpg)

I assume the space is to give room for the inserter.

Just below the header. It takes too much space.
![Screenshot 2021-12-03 at 09 31 03](https://user-images.githubusercontent.com/5323259/144570782-b4119d8d-a5a0-4502-8c69-d0c56d7ce3bf.jpg)

**Expected behavior**
A much thinner gap below the header. Should be the same size as the other gaps between the below paragraph blocks.

Between the paragraph blocks.
![Screenshot 2021-12-03 at 09 30 49](https://user-images.githubusercontent.com/5323259/144570754-c7aea698-762b-4d99-a504-1def5a0d2ca2.jpg)

Local test site running Desktop Server and Chrome.

**Comment by kjellr** [comment]
Thanks for the report. That spacing is intentional, and is applied within the Header template part. Controls to modify that spacing do not appear though, which is something we should fix upstream. I've opened a PR here: 

https://github.com/WordPress/gutenberg/pull/37105

The only other weird bit is that it's a bit hard to discover that the margin is coming from within the template part block. When you select it, the margin sits outside the selection. I've opened https://github.com/WordPress/gutenberg/issues/37106 upstream to track that issue. 

I suggest we leave this issue open until the first PR is resolved.
